#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("step_x", int_t, 0, "Set the number of extract points", 2, 0, 20)
gen.add("step_y", int_t, 0, "Set the number of extract points", 2, 0, 20)
exit (gen.generate (PACKAGE, "jsk_pcl_ros", "ResizePointsPublisher"))
